frontend/src/app/applications/permit-application-view/temp-outfitter-details.component.html

```diff
@@ -48,7 +48,7 @@ <h2 class="app-item-label padding-right-1">Applicant information</h2>
         <span *ngIf="application.applicantInfo.fax.extension"> ext. {{ application.applicantInfo.fax.extension }}</span>
       </span>
       <span class="display: block" *ngIf="application.applicantInfo.website">
-        Website: <a href="{{application.applicantInfo.website | UrlPipe }}" target="_blank">{{application.applicantInfo.website}}</a>
+        Website: <a title="WARNING: This link will open a new window or tab to an applicant-provided URL" href="{{application.applicantInfo.website | UrlPipe }}" target="_blank">{{application.applicantInfo.website}}</a>
```

Comment:
Tagging @jstrothman on this one. I think this is an anti-pattern because it's providing more information to screen reader users than to non-screen reader users. If this warning is relevant, it should be made available to everyone. I think there is a valid use of `title` here to provide extra context, but maybe it should be something like "visit the URL provided by the applicant"?

Reply:
Agreed that if the information is important, the `title` attribute is problematic. Some screenreaders ignore the `title` attribute completely, e.g. VoiceOver, and the `title` attritbute is unavailable to keyboard-only users. 
Recommendation: make text visible in the screen, in parentheses following the link: "(URL provided by applicant will open in new window)"